functional-tests/features/testing-api/testing_headers_setting.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Test setting headers
+    As a kakunin user
+    I want to set the headers
+@wip
+    Scenario: Setting http headers
+        Given I set request headers:
```

Comment:
Do we have some endpoint to test if this works? Content-type test is required for this endpoint to work?

Reply:
i will add endpoints for testing this functionality